crates/apub/src/activities/create_or_update/post.rs

```diff
@@ -38,7 +38,7 @@ use url::Url;
 
 impl CreateOrUpdatePage {
   pub(crate) async fn new(
-    post: ApubPost,
+    post: &ApubPost,
```

Comment:
Could be a good place for an `Rc`/`Cow`.

Reply:
Rc/clone is not enough to avoid cloning in this case because into_json only works on plain ApubPost.

I think this shouldn't be changed to &ApubPost because it's better and more normal for the caller to decide whether or not to clone.

https://rust-lang.github.io/api-guidelines/flexibility.html#caller-decides-where-to-copy-and-place-data-c-caller-control